docs/monitors/heroku-metadata.md

```diff
@@ -0,0 +1,31 @@
+<!--- GENERATED BY gomplate from scripts/docs/templates/monitor-page.md.tmpl --->
+
+# heroku-metadata
+
+Monitor Type: `heroku-metadata` ([Source](https://github.com/signalfx/signalfx-agent/tree/master/pkg/monitors/heroku))
+
+**Accepts Endpoints**: **Yes**
+
+**Multiple Instances Allowed**: Yes
+
+## Overview
```

Comment:
This will need docs on how to enable the metadata in Heroku.

Reply:
Added docs